Harvest.Web/appsettings.json

```diff
@@ -1,10 +1,28 @@
 {
-  "Logging": {
-    "LogLevel": {
+  "Serilog": {

```

Comment:
I'm getting flashbacks of Nlog/Log4Net here it's burning my eyeballs 🔥

Reply:
What, you prefer strongly-typed code over json config?

Reply:
ha yup this isn't Java Spring! 😁